Gems/Atom/Feature/Common/Assets/seedList.seed

```diff
@@ -265,6 +265,13 @@
             },
             "platformFlags": 255,
             "pathHint": "passes/splashscreenpassrequest.azasset"
+        },
+        {
+            "assetId": {
+                "guid": "{34C75C3F-68EB-5FA7-A2A5-F942D17C0922}"
+            },
+            "platformFlags": 255,
+            "pathHint": "passes/lowendrenderpipeline.azasset"
```

Comment:
For Carbonated this needs to be switched to MobilePipeline instead of LowEnd

Reply:
This is for development branch.